notebooks/official/sdk/SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -0,0 +1,941 @@
+{
```

Comment:
<p> </p><p> </p><h2>Launch a Training Job to Create a Model -&gt; Run the training job</h2><p> </p><p>Once we have defined your training script, we will create a model. -&gt; Next, run the training job to train the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>